Makefile

```diff
@@ -99,3 +109,20 @@ pdf:
 		-fmarkdown-implicit_figures \
 		-V geometry:margin=1in \
 		-o $(OUTPUT_DOC)/README.pdf
+
+
+# using the environment variable directly to login against the container registry, while the username
+# and container registry hostname are regular Makefile variables, please, overwrite them as needed.
+registry-login:
+	echo "$$GITHUB_TOKEN" | \
+		ko login "$(REGISTRY_HOSTNAME)" --username=$(REGISTRY_USERNAME) --password-stdin
+
+build-image:
+	ko publish --base-import-paths ./cmd/$(IMGCTRL) 
+
+install:
+	helm template \
+		--namespace="shipwright-build" \
```

Comment:
It makes sense to change the default namespace but I have to ask you to also update [here](https://github.com/shipwright-io/image/blob/main/infra/starter/starter.go#L57) and in the README.md. For example [this](https://github.com/shipwright-io/image/blob/main/README.md?plain=1#L71) is a place where we should also change to `shipwright-build`.

Reply:
And what about using a `make target NAMESPACE="..."` type of construction? So we can let a reasonable default namespace configured, but users could overwrite as needed.

Reply:
Sounds even better

Reply:
Cool! Using the latest changes you can run it like this, for instance:
```bash
make install KO_DOCKER_REPO="ghcr.io/otaviof" NAMESPACE="shipwright-build"
```
By default, the namespace is `shipwright-build`, for the record.